app/Exceptions/DockerNotAvailableException.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Exceptions;
+
+use App\Shell\Environment;
+use App\Shell\Shell;
+use Exception;
+
+class DockerNotAvailableException extends Exception
+{
+    public function render($request = null): void
+    {
+        $console = app('console');
+        $shell = app(Shell::class);
+        $environment = app(Environment::class);
+
+        $console->line('');
+        $console->line($shell->formatErrorMessage('Docker is not available.'));
+        $console->line('');
+
+        if ($environment->isLinuxOs()) {
+            $console->line('Verify that the Docker service is running:');
+            $console->line('  sudo systemctl status docker');
+            $console->line('Start the Docker service:');
+            $console->line('  sudo systemctl start docker');
+
+            if (! $environment->userIsInDockerGroup()) {
```

Comment:
I'm not crazy about this logic in the exception. However, I decided not to throw a unique exception when a user is not in the docker group because it's possible that some environments don't require it. I wanted this message to be a suggestion for why the `docker info` command is failing, not a requirement to use Takeout.

Reply:
Totally get that.